models/demos/patchtst/tt/optimized/multicore.py

```
@@ -0,0 +1,203 @@
+def parallel_channel_processing(
+    config,
+    past_values,
+    device,
+    multicore_config,
+    process_fn,
+    *,
+    parameters,
+):
+    batch_size, sequence_length, num_channels = past_values.shape
+
+    channel_core_ranges = multicore_config.get_core_range_for_channels(num_channels)
```

Ruff (0.14.8)

Local variable `channel_core_ranges` is assigned to but never used

Remove assignment to unused variable `channel_core_ranges`

(F841)